rust-maven-plugin/src/main/java/io/questdb/maven/rust/CargoBuildMojo.java

```diff
@@ -161,8 +159,8 @@ private String getName() {
         return components[components.length - 1];
     }
 
-    private File getTargetDir() {
-        return new File(new File(new File(project.getBuild().getDirectory()), "rust-maven-plugin"), getName());
+    private File getTargetDir(String name) {
+        return new File(new File(new File(project.getBuild().getDirectory()), "rust-maven-plugin"), name);
```

Comment:
Revert this change: There's no need to parameterize this as the only thing that's ever  passed in is `getName()`. Simpler code is better.

Reply:
It was just a micro optimization as getName() does a string split and gets called twice now that ManualOutputRedirector uses it.  But I agree simplicity > micro optimizations in this case.